awaitwhat/utils/helpers.py

```diff
@@ -0,0 +1,15 @@
+import asyncio
+import signal
+import sys
+
+from .. import dot
+
+
+def signal_handler(signalNumber, frame):
```

Comment:
Nitpick: for information only, the more Pythonic way based on [PEP8](https://www.python.org/dev/peps/pep-0008/) is to use `signal_number` rather than `signalNumber`